ush/config_defaults.yaml

```diff
@@ -1755,10 +1745,138 @@ task_make_lbcs:
+      global_albedo4.1x1.grb: '{{ workflow.FIXam }}/global_albedo4.1x1.grb'
+      global_zorclim.1x1.grb: '{{ workflow.FIXam }}/global_zorclim.1x1.grb'
+      global_tg3clim.2.6x1.5.grb: '{{ workflow.FIXam }}/global_tg3clim.2.6x1.5.grb'
+      sfc_emissivity_idx.txt: '{{ workflow.FIXam }}/global_sfc_emissivity_idx.txt'
+      solarconstant_noaa_an.txt: '{{ workflow.FIXam }}/global_solarconstant_noaa_an.txt'
+      global_o3prdlos.f77: '{{ workflow.FIXam }}/ozprdlos_2015_new_sbuvO3_tclm15_nuchem.f77'
+    lateral_boundary_conditions:
+      interval_hours: 1 
+      offset: 3 
+      path: '{{ task_run_fcst.fv3.rundir }}/INPUT/{{ nco.NET_default }}.{{ cycle }}{{ "DOT_ENSMEM" | env  }}.gfs_bndy.tile{{ constants.TILE_RGNL }}.f{{ task_run_fcst.fv3.lateral_boundary_conditions.offset }}.nc' 
+    length: 12
+    model_configure:
+      update_values:
+        total_member: 1
+        start_year: !int '{{ cycle.year }}'
+        start_month: !int '{{ cycle.month }}'
+        start_day: !int '{{ cycle.day }}'
+        start_hour: !int '{{ cycle.hour }}'
+        start_minute: !int '{{ cycle.minute }}'
+        start_second: !int '{{ cycle.second }}'
+        nhours_fcst: !int '{{ workflow.FCST_LEN_HRS }}'
+        fhrot: 0
+        run_continue: false
+        ens_sps: false
+        dt_atmos: ""
+        calendar: julian
+        memuse_verbose: false
+        restart_interval: 0
+        output_1st_tstep_rst: false
+        write_dopost: false
+        ideflate: 0
+        nbits: 0
+        ichunk2d: -1
+        jchunk2d: -1
+        ichunk3d: -1
+        jchunk3d: -1
+        kchunk3d: -1
+        itasks: 1
+        quilting: true
+        write_groups: !int '{{ task_run_fcst.WRTCMP_write_groups }}'
+        write_tasks_per_group: !int '{{ task_run_fcst.WRTCMP_write_tasks_per_group }}'
+        num_files: 2
+        filename_base: 'dyn phy'
+        output_file: 'netcdf netcdf'
+        output_fh: '1  -1'
+        nsout: -1
+        output_grid: '{{ task_run_fcst.WRTCMP_output_grid }}'
+    namelist:
+      base_file: "{{ user.PARMdir }}/input.nml.FV3"
+      update_values:
+        atmos_model_nml:
+          blocksize: !int '{{ task_run_fcst.BLOCKSIZE }}'
+          ccpp_suite: '{{ workflow.CCPP_PHYS_SUITE }}'
+        fv_core_nml:
+          target_lon: !int '{{ grid_params.LON_CTR }}'
+          target_lat: !int '{{ grid_params.LAT_CTR }}'
+          nrows_blend: !int '{{ grid_params.HALO_BLEND }}'
+          stretch_fac: !int '{{ grid_params.STRETCH_FAC }}'
+          npx: !int '{{ grid_params.NX + 1 }}'
+          npy: !int '{{ grid_params.NY + 1 }}'
+          layout:
+            - !int '{{ task_run_fcst.LAYOUT_X }}'
+            - !int '{{ task_run_fcst.LAYOUT_Y }}'
+          bc_update_interval: !int '{{ task_get_extrn_lbcs.LBC_SPEC_INTVL_HRS }}'
+        gfs_physics_nml:
+          kice: !int '{{ 9 if workflow.SDF_USES_RUC_LSM else 2 }}'
+          print_diff_pgr: false
+        nam_sfcperts: {}
+        nam_sppperts: {}
```

Comment:
UW schema-validation error
```
[2024-11-07T14:32:02]    ERROR Error at fv3 -> namelist -> update_values -> nam_sfcperts:
[2024-11-07T14:32:02]    ERROR   {} should be non-empty
```

Reply:
I previously had a fail run where `nam_sfcperts` and `nam_sppperts` were missing, but were required arguments for the test. I added these into the `config_defaults` as empty dictionaries since I wasn’t sure what the key / value pairs would look like and if they would be a dict. The test is failing for both of these keys because the value cannot be an empty dict.